hpi-swt2/lecture-portal
# Feature/#83 files have authorship timestamps

# Resolves #153

This PR adds the UploadedFile-Author-Assosiation to the DB (added a migration) and to rails (created Assosiation: `has_many, belongs_to`).

Acceptance Criteria:
- [x] Any file will have an association to the uploaders account
- [x] it can be determined whether a file belongs to a student or to a lecturer (example code: `file.author.is_student`)
- [x] Any file will have an time stamp of the upload time association to it


Testing steps:
- execute automated tests
- check manually in the rails console (`rails c`)

Steps needed for migration:
- `bundle exec rails db:migrate RAILS_ENV=development && bundle exec rails db:migrate RAILS_ENV=test`

---

Definition Of Done:
